backend/Ript-Fitness-Backend/src/main/java/com/riptFitness/Ript_Fitness_Backend/web/dto/SocialPostDto.java

```diff
@@ -0,0 +1,22 @@
+package com.riptFitness.Ript_Fitness_Backend.web.dto;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.riptFitness.Ript_Fitness_Backend.domain.model.AccountsModel;
+import com.riptFitness.Ript_Fitness_Backend.domain.model.SocialPostComment;
+
+public class SocialPostDto {
```

Comment:
We could consider changing the fields in the DTO to private for encapsulation? / preventing invalid data